speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp

```diff
@@ -468,9 +468,9 @@ void cGrBoard::DispIndicatorText(tTextData* p_data)
     if (!p_data) return;
 
     GfuiDrawString(
-        p_data->Text, normal_color_, GFUI_FONT_LARGE_C, 
-        rightAnchor * p_data->ScrPos.X,
-        TOP_ANCHOR  * p_data->ScrPos.Y);
+        rightAnchor * p_data->ScrPos.X, 
+        TOP_ANCHOR * p_data->ScrPos.Y, 
```

Comment:
```suggestion
        rightAnchor* p_data->ScrPos.X, 
        TOP_ANCHOR* p_data->ScrPos.Y, 
```

Reply:
These are not pointers and I will keep whitespace between arithmetic operations